tests/e2e/cluster/up.sh

```diff
@@ -58,7 +58,7 @@ main() {
 	# Untaint the node so that pods can be scheduled on it.
 	for role in master control-plane; do
 		kubectl taint nodes "$(hostname)" \
-			"node-role.kubernetes.io/$role:NoSchedule-"
+			"node.kubernetes.io/$role:NoSchedule-"
 	done
```

Comment:
```suggestion
			"node-role.kubernetes.io/$role:NoSchedule-"
	done
	kubectl label node "$(hostname)" node.kubernetes.io/worker=
```
As kubeadm will still apply the node-role.kubernetes.io taint, we need to keep removing it and add the new label in addition @kartikjoshi21. This should unblock the e2e test pipeline.

Reply:
Thanks @katexochen . I have added this change.